main.nf

```diff
@@ -11,74 +11,58 @@
 
 nextflow.enable.dsl = 2
```

Comment:
you forgot to left align `nf-core/rnaseq` on line 4

Reply:
Yup. I think this is how it is in the pipeline template [here](https://github.com/nf-core/tools/blob/dff9958cfb5b512abe62983ebf3574d8206bb433/nf_core/pipeline-template/main.nf#L4). But we can change it there too.

Reply:
How does this look? https://github.com/nf-core/rnaseq/pull/622/commits/db9d18f81460d31d6d30212fa4835fa932d957b0

Reply:
Added some appropriate links too whilst I was there :)